.github/workflows/check-pr-title.yaml

```diff
@@ -0,0 +1,31 @@
+name: "Check PR title"
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize]
+
+jobs:
+  check-title:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if the PR title is well dressed
+        env:
+          PR_TITLE: ${{ github.event.pull_request.title }}
+          CONV: '(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)(\([\w\-\.]+\))?!?'
+          ROX: '(ROX-[0-9]+,)*ROX-[0-9]+'
+          TEXT: ': .+'
+          LENGTH: '70'
+        run: |
+          # Either conventional or ROX-nnn prefix followed by ': ' and random text:
+          REGEX="(($CONV)|($ROX))($TEXT)"
+          if ! printf %s "$PR_TITLE" | grep --perl-regexp --line-regexp "$REGEX"
```

Comment:
Curious: why `printf %s "$PR_TITLE"` and not `echo -nE "$PR_TITLE"`?

Reply:
No particular reason, should be the same.